src/app/core/services/fss-search-filter.service.ts

```diff
@@ -15,11 +16,11 @@ export class FssSearchFilterService {
 
   constructor() { }
 
-  getFilterExpression(fssSearchRows: FssSearchRow[],groupings : RowGrouping[]) {
-    
-    var filter='';
+  getFilterExpression(fssSearchRows: FssSearchRow[], groupings: RowGrouping[]) {
 
-    for(var rowIndex=0; rowIndex < fssSearchRows.length; rowIndex++) {
+    var filter = '';
+
+    for (var rowIndex = 0; rowIndex < fssSearchRows.length; rowIndex++) {
```

Comment:
let not var... i wont repeat this for every instance just always use by preference `const` first then `let` if `const` doesn't work and basically never use `var` unless you have no choice.